ci/prep_and_tag_release.sh

```diff
@@ -47,9 +46,30 @@ sed -i -e "s|\(fitbenchmarking/Build%20and%20Publish\)[^\?]*|\1/${tag}|" \
 
 git add README.md
 git commit -m "Prepare release: ${tag}"
+
+# Update citation file if this is not a pre-release
+final_release_regex="^v[0-9]+\.[0-9]+\.[0-9]+$"
+if [[ $tag =~ $final_release_regex ]]; then
+    sed -i -e "s|^version: [0-9.]*|version: ${tag}|" \
+        -e "s|^date-released: '.*'$|date-released: '$(date -I)'|" \
+        CITATION.cff
+
+    git add CITATION.cff
+    git commit -m "Update version in CITATION.cff (${tag})"
+fi
+
 git tag -a "${tag}" -m "Tag generated by prep_and_release.sh"
 
+# Revert README so it points to latest in repo
 mv README.md.bak README.md
 
 git add README.md
 git commit -m "Reset README.md to follow master"
+
+# Give instructions for finishing release
+if [[ $tag =~ $final_release_regex ]]
+    echo "Please cherry-pick the update version commit to master."
+    echo "> git switch master"
+    echo "> git cherry-pick ${tag}"
+    echo "> git push"
+fi
```

Comment:
@tyronerees I'm not sure whether this is fine -- (It's probably going to be one of us doing it anyway) or whether I should make it a branch and PR process.